### PR TITLE
don't delegate to SymPy on flint series exception

### LINKED ISSUE #47: don't delegate to SymPy on flint series exception
(I have to stack things up here to not forget them...)
Somewhere in symgenine.py this exception is not caught:
```
In [2]: from symengine import *

In [3]: x=Symbol('x')                      

In [4]: series(1/cos(x/log(x)), x, 0)
Out[4]: 
         2            4   
        x          5⋅x         ⎛ 6⎞
1 + ───────── + ────────── + O⎝x ⎠
         2            4       
    2⋅log (x)   24⋅log (x)  

In [5]: series(1/cos(x/log(x)), x, 0, method='symengine')
Exception (fmpq_poly_log_series). Constant term != 1.

Abort caught. Printing stacktrace:

Traceback (most recent call last):

Done.
Aborted
```
This SymPy test case is also interesting because, although univariate and having `log(x)`, SymPy solves it by leaving the log as were it another symbol. This shows the cases nomally throwing exceptions with `URatP...` can be treated by other means.

### PR DESCRIPTION
**Pull request by rwst**
fixes #47

### CODE REVIEW

**Comment by rwst** [comment]
"Error fetching appveyor.yml from the repository. Make sure AppVeyor is authorized to access your gitHub account: A task was canceled."
How can the job be restarted (without new commit)?

**Comment by isuruf** [comment]
You'll have to first log in to appveyor and then select `isuruf` as the team to restart the build. I've restarted the build here, https://ci.appveyor.com/project/isuruf/symengine-py-l1jmr/build/122

File: symengine/lib/symengine_wrapper.pyx [hunk 1]
```diff
@@ -543,8 +543,7 @@ def series(ex, x=None, x0=0, n=6, method='sympy', removeO=False):
     try:
         umap = deref(symengine.series(_ex.thisptr, X, N)).as_dict()
     except RuntimeError:
-        from sympy import series as sy_series
-        return sy_series(_ex._sympy_(), _x._sympy_(), x0, n)
+        raise
```
Comment (isuruf) [inline symengine/lib/symengine_wrapper.pyx]: You can remove the `try except` statements

**Comment by rwst** [comment]
Okay, after the hash update all is green.

**Comment by isuruf** [comment]
+1

**Comment by rwst** [comment]
Thanks.

**Pull request merged**